src/persist-client/src/impl/state.rs

```diff
@@ -188,6 +179,7 @@ where
         )
     }
 
+    #[allow(dead_code)]
```

Comment:
we're planning on ripping all this out in the followup?

Reply:
yes!